apps/dashboard/src/data/projects/reducer.js

```diff
@@ -46,8 +48,44 @@ const isSaving = ( state = false, action ) => {
 	return state;
 };
 
+const isDraftSaved = ( state = false, action ) => {
```

Comment:
Like mentioned before, I believe both `isDraftSaved` and `isPublishSaved` could be replaced by a single `status` reducer.  

Note: I think if we go this way this one should best be 'keyed', similar to `items`.